barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/goblin_ultra_circuit_builder.test.cpp

```diff
@@ -45,7 +45,7 @@ TEST(GoblinUltraCircuitBuilder, CopyConstructor)
     GoblinUltraCircuitBuilder duplicate_circuit_constructor{ circuit_constructor };
 
     EXPECT_EQ(duplicate_circuit_constructor.get_num_gates(), circuit_constructor.get_num_gates());
-    EXPECT_EQ(duplicate_circuit_constructor.selectors.get(), circuit_constructor.selectors.get());
+    // EXPECT_EQ(duplicate_circuit_constructor.get_selectors(), circuit_constructor.get_selectors());
```

Comment:
Could you delete this and add a TODO to reinstate selector checking after you're finished with blocks?

Reply:
Thanks. Updated this to check equality of the two builders which is better than what it was before